mem-01-b.xml

```diff
@@ -307,6 +342,7 @@ The suffix {-Daq:n} marks the target.</column>
       <column name="notes_sv"></column>
       <column name="notes_ru"></column>
       <column name="notes_zh_HK"></column>
+      <column name="notes_pt">utensílios de cozinha</column>
```

Comment:
Generally, the other-language notes should be translations of the English notes. Is this needed to distinguish the "spoon" meaning of "colher" from other meanings? Can it be made part of the definition?